src/modules/samples/rust/src/lib.rs

```diff
@@ -1,136 +1,64 @@
-use libc::{c_int, c_char ,c_void};//, c_void
+// Copyright (c) Microsoft Corporation. All rights reserved..
+// Licensed under the MIT License.
+
 use std::ffi::CStr;
 use std::ffi::CString;
-use std::ptr::NonNull;
+use std::os::raw::{c_char, c_int, c_uint, c_void};
 
-mod sample;
+pub mod sample;
 
 // Constants and Method headers were generated by bindgen
 // (https://github.com/rust-lang/rust-bindgen)
 pub const MMI_OK: u32 = 0;
-pub type MMI_HANDLE = *mut ::std::os::raw::c_void;
-pub type MMI_JSON_STRING = *mut ::std::os::raw::c_char;
+pub type MMI_HANDLE = *mut c_void;
+pub type MMI_JSON_STRING = *mut c_char;
 
 // MSFT change: Added the #[no_mangle] annotation and defined the methods
```

Comment:
Is this comment necessary? The credit above given to `bindgen` seems sufficient.